libs/shared/account/account/src/lib/account.manager.ts

```diff
@@ -42,4 +43,9 @@ export class AccountManager {
     });
     return uid.toString('hex');
   }
+
+  async getAccounts(uids: string[]) {
+    const bufferUids = uids.map((uid) => uuidTransformer.to(uid));
+    return getAccounts(this.db, bufferUids);
```

Comment:
Do the uids come out as buffers as well?

Reply:
They do. It's handled in [the cleanup script](https://github.com/mozilla/fxa/blob/b04c3cbaa5273a5a400f739fff6ef15db475043c/packages/fxa-auth-server/scripts/cleanup-partial-firestore-customers.ts#L77-L79). I figured it'd be better to return the full array of accounts as is, to make this function more reusable.

```
const accounts = await this.accountManager.getAccounts(uids);
const accountUids = accounts.map((account) =>
  uuidTransformer.from(account.uid)
);
```